File: terraimmerse/world/entity/PlayerEntity.py
import glm

class PlayerEntity:
    def __init__(self):
        self.pos = glm.vec3(0, 0, 0)
        self.yaw = 0
        self.pitch = 0
    def move(self, x, y, z):
        self.pos += glm.vec3(x, y, z)
    def setPos(self, x, y, z):
        self.pos = glm.vec3(x, y, z)
    def getPos(self):
        return self.pos

File: piEdition/src/main/java/net/terraimmerse/world/entity/PlayerEntity.java
package net.terraimmerse.world.entity;

import org.joml.Vector3f;

public class PlayerEntity {
    public Vector3f pos;
    public float yaw;
    public float pitch;
    public PlayerEntity(){
        this.pos = new Vector3f(0.0F,0.0F,0.0F);
        this.yaw=0.0F;
        this.pitch=0.0F;
    }
    public void move(float x, float y, float z){
        this.pos.x+=x;
        this.pos.y+=y;
        this.pos.z+=z;
    }
    public void setPos(float x, float y, float z){
        this.pos = new Vector3f(x, y, z);
    }
    public Vector3f getPos(){
        return this.pos;
    }
}
